build/common.project.props

```diff
@@ -124,7 +124,8 @@
     <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
     <VsixOutputDirName>VS15</VsixOutputDirName>
     <CodeAnalysisRuleSet Condition="'$(CodeAnalysisRuleSet)' == ''">$(BuildCommonDirectory)NuGet.ruleset</CodeAnalysisRuleSet>
-    <EnforceCodeStyleInBuild Condition=" '$(EnforceCodeStyleInBuild)' != '' " >true</EnforceCodeStyleInBuild>
+    <EnforceCodeStyleInBuild Condition="'$(EnforceCodeStyleInBuild)' == ''">true</EnforceCodeStyleInBuild>
```

Comment:
If I am reading [document](https://learn.microsoft.com/dotnet/fundamentals/code-analysis/overview?tabs=net-8#enable-on-build) correctly, it looks like we are relying on the default severity set by the .NET SDK Analyzers. I am okay with this but just wanted to share that there is an option to set severity levels of these rules. I think we should start with default severity set by the analyzers and adjust as we progress.

Reply:
We have an .editorconfig file with a bunch of values already too.